.travis.yml

```diff
@@ -40,15 +40,13 @@ before_install:
       brew uninstall xctool;
       brew install xctool --HEAD;
       brew install homebrew/versions/glfw3;
-      brew install libusb;
     fi
 
   # Install linux required packages
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/test;
       sudo apt-get update;
-      sudo apt-get install -qq build-essential xorg-dev libgl1-mesa-dev libglu1-mesa-dev libglew-dev libglm-dev;
-      sudo apt-get install -qq libusb-1.0-0-dev;
+      sudo apt-get install -qq build-essential xorg-dev libgl1-mesa-dev libglu1-mesa-dev libglew-dev libglm-dev libudev-dev;
```

Comment:
Should the -qq package appear in installation.md ?

Reply:
Yes, libudev-dev should be added to the installation.md